app/build.gradle.kts

```diff
@@ -81,6 +82,7 @@
         buildConfig = true
         viewBinding = true
         compose = true
+        dataBinding = true
```

Comment:
If you don't have a specific reason to use data binding here, I can help refactor the PR to avoid data binding altogether since the project doesn't already use it

Reply:
I can switch to view binding. I think going back to findViewById is not desireable since it has no compile time verification.

Reply:
I believe viewBinding would be perfect here since it's already enabled in the project. 
I also agree that `findViewById` is definitely not desirable

Reply:
Using Jetpack Compose is our current preferred way for UI

Reply:
Okay just added the commit to move to view binding